src/pdsparsemat.jl

```diff
@@ -84,22 +84,22 @@ end
 
 function X_A_Xt(a::PDSparseMat, x::StridedMatrix)
     z = x*sparse(chol_lower(a.chol))
-    A_mul_Bt(z, z)
+    z * z'
 end
 
 
 function Xt_A_X(a::PDSparseMat, x::StridedMatrix)
-    z = At_mul_B(x, sparse(chol_lower(a.chol)))
-    A_mul_Bt(z, z)
+    z = x' * sparse(chol_lower(a.chol))
```

Comment:
Note that the direct translation from the `A_op_B` style would be to use `transpose(x)`, since `x'` is the adjoint. You were using `transpose` in src/pdmat.jl, probably best to be consistent.

Reply:
Thanks for catching that.  I just pushed a commit with those changes.  I hope it is ready to go now.